packages/wallets/src/components/Base/Tabs/Tab.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC, ReactNode } from 'react';
+
+type TabProps = {
+    children: ReactNode;
+    icon?: ReactNode;
+    title: string;
+};
+
+const Tab: FC<TabProps> = ({ children }) => <>{children}</>;
```

Comment:
Merge this with TabTitle and name it `Tab`. This component is actually just forwarding the props forward